README.md

```diff
@@ -1,12 +1,248 @@
-# ICGC - Software Engineering Blog
+<!--Sources-->
+<!--https://blog.hubspot.com/marketing/what-is-a-pillar-page-->
+<!--https://thewritepractice.s3-us-west-1.amazonaws.com/Blog+Post+Template+Worksheet.pdf-->
+<!--https://www.verblio.com/blog/blog-post-length#:~:text=The%20ideal%20paragraph%20size%20for,that%20number%20can%20fluctuate%20wildly.-->
+<!--https://thewritepractice.com/formatting-tips/.-->
+<!--https://blog.hubspot.com/marketing/how-long-should-your-blog-posts-be-faq-->
+<!--https://www.wix.com/blog/2021/06/how-long-should-a-blog-post-be/#:~:text=After%20in%2Ddepth%20industry%20research,sweet%20spot%20right%20at%202%2C450.-->
 
-Although not mandatory, you might want to run a local instance of the blog to verify that your blog post content renders properly.
+# :newspaper: OICR Engineering Blog 
 
-## Instructions to run a local instance of Jekyll
+<img src="images/OICRHero.jpg">
 
-First clone this repo
+<br />
 
-Then cd into the repo and run:
+<p align="center"><a href="#book-writing-guide">Writing Guide</a>  |  <a href=#computer-running-a-local-instance>Running a Local Instance</a>  |  <a href=#ballot_box-submitting-a-blog-post>Submitting a Blog Post</a>
+
+We are the OICR team of software engineers, infrastructure professionals, and bioinformaticians developing big data tools to help in the fight against cancer. Here we share our successes, failures, curiosities and encouragement for open technical discussion with you, the reader. Welcome!
+
+<br />
+
+# :date: Bi-weekly Content Calendar 
```

Comment:
Let's move the writing schedule out of the readme, its difficult to maintain here and prone to regular changes. Its not what people look for when landing on the repo. As discussed offline, scheduling posts and tracking their content through github issues is a systme that has worked in the past.

Reply:
Done!